### PR TITLE
[yang] Support ASIC/SDK health event

### PR DESCRIPTION
**Pull request by stephenxs**
#### Why I did it

Support ASIC/SDK health event: yang model update
- Suppress ASIC/SDK health event configuration
- New event

##### Work item tracking
- Microsoft ADO **(number only)**:

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@qiluo-msft  @dgsudharsan  can you please help to review the PR?

**Review by Junchao-Mellanox — APPROVED** [review]

**Review by dgsudharsan — CHANGES_REQUESTED** [review]

**Review by dgsudharsan — APPROVED** [review]

File: src/sonic-yang-models/yang-models/sonic-suppress-asic-sdk-health-event.yang [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                leaf severity {
+                    type enumeration {
+                        enum fatal;
+                        enum warning;
+                        enum notice;
+                    }
+                    description "Severity of the ASIC/SDK health event to suppress";
+                }
+
+                leaf max_events {
```
[the remark this answers is not in the document]
Comment (stephenxs) [reply src/sonic-yang-models/yang-models/sonic-suppress-asic-sdk-health-event.yang:42]: Fixed.

**Comment by stephenxs** [comment]
Seems to be a dependency issue.
The `sonic-yang-mgmt` test depends on the latest `sonic-yang-model` in this PR. But at the time when the `sonic-yang-mgmt` test was running the latest `sonic-yang-model` python wheel had not been installed.
Trying to find a quick way to resolve the dependency issue for now but if there isn't I would like to move `sample_config_db.json` into another PR depending on this one.

**Comment by stephenxs** [comment]
> Seems to be a dependency issue. The `sonic-yang-mgmt` test depends on the latest `sonic-yang-model` in this PR. But at the time when the `sonic-yang-mgmt` test was running the latest `sonic-yang-model` python wheel had not been installed. Trying to find a quick way to resolve the dependency issue for now but if there isn't I would like to move `sample_config_db.json` into another PR depending on this one.

Fixed.

**Comment by stephenxs** [comment]
@qiluo-msft @prsunny Could you help to review and merge the PR? This is the yang model of ASIC/SDK health event. Thanks.

**Comment by bingwang-ms** [comment]
@kperumalbfn @prabhataravind Can you please help review this change?

**Comment by liat-grozovik** [comment]
@zhangyanzhao @qiluo-msft could you please help to approve it on the coming yang meeting? if not, i suggest to merge it as we need it for 202405.

**Review by prsunny — APPROVED** [review]

**Comment by prsunny** [comment]
@kperumalbfn , please merge once you signoff. Yang change is approved by Sudharsan.

**Review by kperumalbfn — APPROVED** [review]

**Pull request merged**